### ISSUE TITLE
TypeError: issubclass() arg 1 must be a class

### ISSUE THREAD

**Issue by feljen** [limdauto/drf_openapi#92]
* DRF OpenAPI version: 1.0.0
* Python version: 3.4.3
* Operating System: Ubuntu (vagrant VM)

Trying to access the URL of the rendered schema produces the following error:

```
Traceback (most recent call last):
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/rest_framework/decorators.py", line 52, in handler
    return func(*args, **kwargs)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/drf_openapi/views.py", line 22, in schema_view
    return response.Response(generator.get_schema(request))
  File "/home/ubuntu/virt/lib/python3.4/site-packages/drf_openapi/entities.py", line 93, in get_schema
    links = self.get_links(None if public else request)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/drf_openapi/entities.py", line 133, in get_links
    link = self.get_link(path, method, view, version=request.version)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/drf_openapi/entities.py", line 187, in get_link
    response_serializer_class)
  File "/home/ubuntu/virt/lib/python3.4/site-packages/drf_openapi/entities.py", line 214, in get_paginator_serializer
    if issubclass(pager, (PageNumberPagination, LimitOffsetPagination)):
TypeError: issubclass() arg 1 must be a class
```

**Comment by tuffnatty** [comment]
Seems to be a duplicate of #84. Try the PR in #85 and see if it fixes your problem.

**Comment by limdauto** [comment]
@feljen this should be fixed in 1.1.0
@tuffnatty thanks for pointing it out
